README.md

```diff
@@ -72,6 +72,8 @@ public class Startup
 
 If you use MinimalApi rather than asp.net core Startup class.
 
+Add package reference for `Xunit.DependencyInjection.AspNetCoreTesting`
+
 ``` bash
```

Comment:
should probably change this to ```sh for consistence
```suggestion
```sh
```

Reply:
nice catch, thanks